src/index.ts

```diff
@@ -92,3 +92,97 @@ export function render(
     })
     .catch(error => showError(el, error));
 }
+
+interface WidgetUpdate {
+  key: string;
+  remove?: string;
+  insert?: any[];
+}
+
+interface WidgetUpdateMessage {
+  type: "update";
+  updates: WidgetUpdate[];
+}
+
+function checkWidgetUpdate(ev: any): WidgetUpdateMessage | null {
+  if (ev.type != "update") {
+    return null;
+  }
+  // TODO: perform more checks and validate the event
```

Comment:
What kind of checks and for what? I don't really know how to work with this TODO.

Reply:
Godd point. I also needed a minute to refamilarize myself with my thought process :).

Essentially, the idea was validate that the event really conforms to the `WidgetUpdateMessage` type, i.e., also checking the `updates` array. With a bit of distance, I'm no longer sure, whether it really is relevant, as the API is purely used internally.

Reply:
Please update the TODO accordingly then.